ngraph/frontend/onnx_import/include/onnx_import/core/tensor.hpp

```diff
@@ -202,6 +219,12 @@ namespace ngraph
                 template <>
                 inline std::vector<float> get_data(const ONNX_NAMESPACE::TensorProto& tensor)
                 {
+                    if (tensor.has_data_location() &&
```

Comment:
Please add a helper function, so this can be replaced to a call to `tensor_has_external_data` instead of repeating the same complex condition multiple times.

Reply:
done, `has_tensor_external_data` added